lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3147,6 +3147,21 @@ bool SPIRVToLLVM::transKernelMetadata() {
       auto SizeMD = ConstantAsMetadata::get(getUInt32(M, EM->getLiterals()[0]));
       F->setMetadata(kSPIR2MD::SubgroupSize, MDNode::get(*Context, SizeMD));
     }
+    // Generate metadata for intel_max_work_group_size
```

Comment:
These metadata don't have `intel_` prefix in the test

Reply:
No, fixed.